Dockerfile

```diff
@@ -48,19 +36,9 @@ RUN \
   && cat *aerospike-tools*.sha256 | cut -d' ' -f1) \
   && echo "$TOOLS_SHA256 *aerospike-tools.tgz" | sha256sum -c -
 
-FROM debian:bullseye-slim as install
```

Comment:
Why remove multi-stage builds? This is the recommended way of separating setup and install steps according to docker.

Reply:
I didn't know this was recommended. I though removing it made the build simpler and the file easier to read. After reading https://docs.docker.com/build/building/multi-stage/ I think it's better to keep a multi stage build to make keeping the final image cleaner, easier. Changed back.